### PR TITLE
multiple grammer changes to Update 030_Motivations.md

### PR DESCRIPTION
**Pull request by chriskucharski**
multiple grammar changes to improve the reading flow.

### CODE REVIEW

File: manuscript/030_Motivations.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
 Motivation -- the first step to learning TDD
 ===========================================
 
-I'm writing this book because I'm a TDD enthusiast. I believe TDD is a huge improvement over other software development disciplines I have used for the same purpose. I believe this is true not only for me, but for many, many other software developers. If so, then everyone should learn TDD and use it, shouldn't they? Yet, in my professional life, I haven't seen the adoption rate to be big enough to justify the claim that TDD is currently in the mainstream.
+I'm writing this book because I'm a TDD enthusiast. I believe TDD is a huge improvement over other software development methodologies I have used to deliver quality software. I believe this is true not only for me, but for many other software developers. Which makes me question, why don't more people learn as use TDD as their software delivery methodology of choice? In my professional life, I haven't seen the adoption rate to be big enough to justify the claim that TDD is currently in the mainstream.
```
Comment (grzesiek-galezowski) [inline manuscript/030_Motivations.md]: learn *as* use TDD -> learn *and* use TDD?

File: manuscript/030_Motivations.md [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
 
-You already have my respect for deciding to pick up a book, rather than building your understanding of TDD on the foundation of urban legends and your own imagination. I am honored and happy you chose this one, no matter if this is your first book on TDD or maybe you've already read, like, nine of them. As much as I really hope you will read this book from cover to cover, I am aware it doesn't always happen. That makes me want to ask you an important question that may help you determine whether you really want to read on: why do you want to learn TDD?
+You already have my respect for deciding to pick up a book, rather than building your understanding of TDD on the foundation of urban legends and your own imagination. I am honored and happy you chose this one, no matter if this is your first book on TDD or one of many you may have read. As much as I really hope you will read this book from cover to cover, I am aware it doesn't always happen. That makes me want to ask you an important question that may help you determine whether you really want to read on: why do you want to learn TDD?
```
Comment (grzesiek-galezowski) [inline manuscript/030_Motivations.md]: "of many you may have read" - I like the wording, however, I feel there should be a distinction between the books the reader may have read and this one which the reader probably just started. Using past tense for both other books and this one sort of feels like the reader finished reading this book. What do you think?

**Comment by grzesiek-galezowski** [comment]
Besides the two comments that I'd like to discuss, I like your changes very much! Can't wait to resolve the outstanding issues and merge it :-)

File: manuscript/030_Motivations.md (hunk 1 above)
Comment (chriskucharski) [inline manuscript/030_Motivations.md]: Good catch!

File: manuscript/030_Motivations.md (hunk 2 above)
Comment (chriskucharski) [inline manuscript/030_Motivations.md]: What do you think about something like this, "I am honored and happy you chose this one, no matter if this is your first book on TDD or one of many you may opened up in your learning endeavors."

Comment (chriskucharski) [inline manuscript/030_Motivations.md]: Or maybe this, "I am honored and happy you chose this one, no matter if this is your first book on TDD or one of many you have opened up in your learning endeavors."

File: manuscript/030_Motivations.md (hunk 1 above)
Comment (chriskucharski) [inline manuscript/030_Motivations.md]: corrected

File: manuscript/030_Motivations.md (hunk 2 above)
Comment (grzesiek-galezowski) [inline manuscript/030_Motivations.md]: I like the last version! Please make the change and I'll merge the pull request.

Comment (chriskucharski) [inline manuscript/030_Motivations.md]: Done!

**Comment by grzesiek-galezowski** [comment]
Here's my way of saying thank you: 5476653f8dca9c2254048141a9071d3e23b3b027 ! I hope you don't mind :-)

**Pull request merged**